src/frontend/views/modules/header/header.pug

```diff
@@ -6,7 +6,7 @@ div
       ul.list-inline.all-centered
         li.logo
           a(href='', ui-sref='home')
-            img.openpaas-logo(ng-src="{{ baseUrl + '/api/themes/' + domainId + '/logo'}}", fallback-src='assets/images/white-logo.svg', src='assets/images/white-logo.svg', alt="{{ 'OpenPaas Logo' | translate }}")
+            img.openpaas-logo(ng-src=`{{logoUrl}}`, fallback-src='assets/images/white-logo.svg', src='assets/images/white-logo.svg', alt="{{ 'OpenPaas Logo' | translate }}")
```

Comment:
```suggestion
            img.openpaas-logo(ng-src=`{{logoUrl}}`, fallback-src='assets/images/white-logo.svg', alt="{{ 'OpenPaas Logo' | translate }}")
```

this should work without the `src` attribute, since we have `ng-src` ?

Reply:
It seems the `src` attributes is used first, then the `ng-src` replaces `src` value with and expression with variables replaced by their value, and `fallback-src` is fired if `ng-src` url errors.
Though I guess removing src would not have much effect.

Reply:
Right `src` is used first and necessary for `fallbackSrc` directive 
But with `restangular` we can catch error so will delete all this unsed code

Reply:
are you sure trick is better than native fallbackSrc ? can you give some pros/cons ?

Reply:
fallbackSrc is a custom directive. And it was used to compensate logo fetching issues.
We fetch logo even if there is no custom logo and in this case theme api already send a fallback image

Reply:
fallbackSrc it works well before, but now we go to the backend to retrieve the downloaded logo and on the browser we first see the default logo appear then we see the correct logo, for that I proposed to delete the directive rather than adding a flag (loading)